cloudos_cli/__main__.py

```diff
@@ -1832,6 +1834,19 @@ def run_bash_job(ctx,
               f'\t\t--job-id {j_id}\n')
 
 
+
+def format_bytes(size):
+    """Convert bytes to human-readable format (e.g., 1.2 MB)."""
+    if size is None:
+        return "-"
+    power = 1024
+    n = 0
+    labels = ['B', 'KB', 'MB', 'GB', 'TB', 'PB']
+    while size >= power and n < len(labels) - 1:
+        size /= power
+        n += 1
+    return f"{size:.1f} {labels[n]}"
+
```

Comment:
Could we move this function (alongside with `ssl_selector` ) into a new script in `cloudos-cli/utils/resources.py` or similar. Wdyt? It will be better organised, right?